### ISSUE TITLE
Help needed for tweepy app

### ISSUE THREAD

**Issue by Shekharnunia** [do-n-khanh/mybadminton#1]
Hello, @do-n-khanh I saw your contribution and the approach toward the direct message problem in the tweepy app. I need some help regarding the same, I would like to talk more about it if you continue it. I will be waiting for your response

**Comment by do-n-khanh** [comment]
yes, how can i help you? 
i only know how to send DM message. 
i am not familiar with different things.

**Comment by Shekharnunia** [comment]
I want to send messages to anyone who follows me, but there is no direct way to do this, and Twitter also not allow to do user stream, it was supporting it but now links are changed and tweepy not supporting it. Do you have any idea about this

**Comment by do-n-khanh** [comment]
please look at the solution here https://github.com/tweepy/tweepy/issues/1081#issuecomment-413770522

**Comment by Shekharnunia** [comment]
Yes, I used this solution to send a message and the message part is working fine. The problem is in getting information about followers

**Comment by saroj1987** [comment]
@do-n-khanh I am facing issue while reading the direct message using tweepy, the on_direct_message fo Stream, never called when a message reached to the account. could you please help me out on this. these are following info i am using. 

python3.7
tweepy is latest installed with https://github.com/tweepy/tweepy/issues/1063

code:
class StdOutListener( StreamListener ):

   ...... 
    def on_data( self, status ):
        print("Entered on_data()")
        print(status, flush = True)
        return True

    def on_direct_message( self, status ):
        print("Entered on_direct_message()")
        try:
            print(status, flush = True)
            return True
        except BaseException as e:
            print("Failed on_direct_message()", str(e))

    def on_error( self, status ):
        print(status)
in main:
        twitter_stream=Stream(auth,StdOutListener())
        print('Stream created...')
        twitter_stream.filter(follow=[user.id_str], is_async=True)

NB: persission is Read,Write and direct message. All keys are good. am able connect and also able to read tweets.
could you please help me out.
